LOAN/LoansTemporal/LoanApplicationsTemporal.rdf

```diff
@@ -201,65 +238,34 @@
 		<skos:definition xml:lang="en">That party which appraises the borrower for the purposes of the loan.</skos:definition>
 	</owl:Class>
 	
-	<owl:ObjectProperty rdf:about="&fibo-loan-tem-app;applicationDate">
-		<rdfs:label xml:lang="en">application date</rdfs:label>
+	<owl:ObjectProperty rdf:about="&fibo-loan-tem-app;hasApplicationDate">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-dt-fd;hasExplicitDate"/>
+		<rdfs:label xml:lang="en">has application date</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-loan-ln-ln;LoanApplication"/>
-		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;Date"/>
+		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;ExplicitDate"/>
 	</owl:ObjectProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-loan-tem-app;applicationPhase">
-		<rdfs:label xml:lang="en">application phase</rdfs:label>
+	<owl:ObjectProperty rdf:about="&fibo-loan-tem-app;hasApplicationPhase">
```

Comment:
This should be a general property for Application not specific to Loans. A LoanApplication will restrict the property to have values from LoanApplicationPhase.

Reply:
Ok - next pass.  I will use a lifecycle property, or maybe we need a loan application life cycle that is proper here.